traits/util/tests/test_trait_documenter.py

```diff
@@ -1,30 +1,66 @@
 # -*- coding: utf-8 -*-
 """ Tests for the trait documenter. """
 
-
+import contextlib
+import io
+import os
+import shutil
+import tempfile
 import textwrap
 import tokenize
 import unittest
+try:
```

Comment:
(suggestion) It might be useful to factor out the import checks into a test_compat.py or test_common.py module.

Reply:
Agreed; I don't see us re-using this one much, but the NumPy check and associated test skipping is used in several places.

Reply:
> I don't see us re-using this one much

Sorry; wasn't looking at the context properly. We do of course use the `mock` check in many places. I was thinking of the Sphinx existence check.

Reply:
I'll open an issue for addressing this.

Reply:
👍

Reply:
> I'll open an issue for addressing this.

#496.